src/components/ok-button/ok-button.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from "react";
+import styles from './ok-button.module.scss'
+
+interface OkButtonProps {
+    text: string;
```

Comment:
Each button should has `onClick` method implementation, otherwise it is not really makes sense to use it